dts/arm/ti/mspm0g1x0x_g3x0x/mspm0g1x0x_g3x0x-pinctrl.dtsi

```diff
@@ -0,0 +1,146 @@
+#include <dt-bindings/pinctrl/mspm0-pinctrl.h>
+
+&pinctrl {
+
+	/omit-if-no-ref/ i2c0_scl_pa1_pull_up: i2c0_scl_pa1_pull_up {
+		pinmux = <MSP_PINMUX(2,MSPM0_PIN_FUNCTION_3)>;
+		input-enable;
+		drive-open-drain;
+	};
+
+	/omit-if-no-ref/ i2c0_sda_pa0_pull_up: i2c0_sda_pa0_pull_up {
+		pinmux = <MSP_PINMUX(1,MSPM0_PIN_FUNCTION_3)>;
+		input-enable;
+		drive-open-drain;
+	};
+
+	/omit-if-no-ref/ i2c1_sda_pa10: i2c1_sda_pa10 {
+		pinmux = <MSP_PINMUX(21,MSPM0_PIN_FUNCTION_8)>;
+		input-enable;
+		bias-high-impedance;
+	};
+
+	/omit-if-no-ref/ i2c1_scl_pa11: i2c1_scl_pa11 {
+		pinmux = <MSP_PINMUX(22,MSPM0_PIN_FUNCTION_8)>;
+		input-enable;
+		bias-high-impedance;
+	};
+
+	/omit-if-no-ref/ i2c1_scl_pb2: i2c1_scl_pb2 {
+		pinmux = <MSP_PINMUX(15,MSPM0_PIN_FUNCTION_4)>;
+		input-enable;
+		bias-high-impedance;
+	};
+
+	/omit-if-no-ref/ i2c1_sda_pb3: i2c1_sda_pb3 {
+		pinmux = <MSP_PINMUX(16,MSPM0_PIN_FUNCTION_4)>;
+		input-enable;
+		bias-high-impedance;
+	};
+
+	/omit-if-no-ref/ i2c1_sda_pa16_pull_up: i2c1_sda_pa16_pull_up {
+		pinmux = <MSP_PINMUX(38,MSPM0_PIN_FUNCTION_4)>;
+		input-enable;
+		bias-pull-up;
+		bias-high-impedance;
+	};
+
+	/omit-if-no-ref/ i2c1_scl_pa17_pull_up: i2c1_scl_pa17_pull_up {
+		pinmux = <MSP_PINMUX(39,MSPM0_PIN_FUNCTION_4)>;
+		input-enable;
+		bias-pull-up;
+		bias-high-impedance;
+	};
+
+	/omit-if-no-ref/ i2c1_scl_pa29_pull_up: i2c1_scl_pa29_pull_up {
+		pinmux = <MSP_PINMUX(4,MSPM0_PIN_FUNCTION_2)>;
+		input-enable;
+		bias-pull-up;
+		bias-high-impedance;
+	};
+
+	/omit-if-no-ref/ i2c1_sda_pa30_pull_up: i2c1_sda_pa30_pull_up {
+		pinmux = <MSP_PINMUX(5,MSPM0_PIN_FUNCTION_2)>;
+		input-enable;
+		bias-pull-up;
+		bias-high-impedance;
+	};
+
+	/omit-if-no-ref/ i2c0_sda_pa28: i2c0_sda_pa28 {
+		pinmux = <MSP_PINMUX(3,MSPM0_PIN_FUNCTION_3)>;
+		input-enable;
+		bias-high-impedance;
+	};
+
+	/omit-if-no-ref/ i2c0_scl_pa31: i2c0_scl_pa31 {
+		pinmux = <MSP_PINMUX(6,MSPM0_PIN_FUNCTION_3)>;
+		input-enable;
+		bias-high-impedance;
+	};
+
+	/omit-if-no-ref/ uart0_tx_pa10: uart_tx_pa10 {
+		pinmux = <MSP_PINMUX(21,MSPM0_PIN_FUNCTION_2)>;
+		drive-strength = <20>;
```

Comment:
Should this be "high" rather than <20> as per dts bindings dts/bindings/pinctrl/ti,mspm0-pinctrl.yaml?

Reply:
You're right, I think it's better to keep the drivestrenght as an int which is 6 or 20.